### ISSUE TITLE
Unexpected value 'CKEditorModule' imported by the module 'AppModule'`

### ISSUE THREAD

**Issue by theunreal** [chymz/ng2-ckeditor#50]
`Uncaught Error: Unexpected value 'CKEditorModule' imported by the module 'AppModule'`

After installing & importing in my app module.

Any ideas?

**Comment by LeonardCModoran** [comment]
Having the same error, when using Webpack. With SystemJS everything is working fine.

**Comment by Nandan108** [comment]
Hi!
I tracked down the error to it's point of origin, in `webpack:///./~/@angular/core/bundles/compiler.umd.js:14038`.

The compiler verifies that each module is truly a module by checking that object provided has a metadata object that is an instance of DecoratorFactory as defined in `webpack:///./~/@angular/core/bundles/core.umd.js.`

The problem happens when the CKEditorModule's metadata object is instead an instance of DecoratorFactory as defined in another file:
`webpack:///./~/ng2-ckeditor/~/@angular/core/bundles/core.umd.js`.

In short, the issue arises from CKEditorModule using its own, separate copy of angular core, when loaded via WebPack. This explains why LeonardCModoran's workaround worked: copying the Ng2CKEditor code into his project ensured that it used the same angular core as the rest of his code.

That's as far as I'll go unfortunately. I haven't studied how to develop and publish shareable components or libraries in Ng2, so I haven't got a clue how to fix this. Hopefully someone can take it from there?

**Comment by chymz** [comment]
Can someone please share a reproduction repo on github ? I'll try to fix this soon, and repo can help some people in #54 

Thanks !

**Comment by chymz** [comment]
Hi,
Can anyone tests latest version (1.1.5) ?
Thanks

**Comment by Nandan108** [comment]
Hi,
The issue happened to me again some time ago for the same reason described above: node_modules/ng2-ckeditor/**node_modules/** was present. After removing node_modules, it worked fine.

Just now I updated to 1.1.5 and it installed just fine, without the incriminated subdir.
I guess you can close this issue.

**Comment by chymz** [comment]
thanks @Nandan108 !
